sympy/printing/fcode.py

```diff
@@ -178,6 +172,32 @@ def _print_Piecewise(self, expr):
                                       "inline operators is not supported in "
                                       "standards earlier than Fortran95.")
 
+    def _print_Assignment(self, expr):
```

Comment:
Because here @bjodah overrided the `_print_Assignment` in `codeprinter` i.e the base class.